### PR TITLE
print error msg when plugin binary is missing

### PR DESCRIPTION
**Pull request by rajatjindal**

### CODE REVIEW

File: src/commands/external.rs [hunk 1]
```diff
@@ -66,7 +67,15 @@ pub async fn execute_external_subcommand(
     )
     .await?;
 
-    let mut command = Command::new(plugin_store.installed_binary_path(&plugin_name));
+    let binary = plugin_store.installed_binary_path(&plugin_name);
+    if !Path::new(binary.as_os_str()).exists() {
+        return Err(anyhow!(
+            "plugin executable {:?} is missing. Try uninstalling and installing the plugin again.",
```
Comment (itowlson) [inline src/commands/external.rs:73]: Use the `common::ui::quoted_path()` function (`anyhow!("plugin executable {} ...", quoted_path(&binary))`) - this ensures consistent formatting and correct rendering across platforms.  (We found that the debug format prints double backslashes on Windows.)

File: src/commands/external.rs [hunk 2]
```diff
@@ -66,7 +67,15 @@ pub async fn execute_external_subcommand(
     )
     .await?;
 
-    let mut command = Command::new(plugin_store.installed_binary_path(&plugin_name));
+    let binary = plugin_store.installed_binary_path(&plugin_name);
+    if !Path::new(binary.as_os_str()).exists() {
```
Comment (itowlson) [inline src/commands/external.rs:71]: `binary` is already a `PathBuf` so has the `exists` method built in - no need to extract the string or construct a `Path`.

**Review by itowlson — CHANGES_REQUESTED** [review]
The content is fine - thanks!  Just a couple of style niggles that it would be good to clear up - sorry.

File: src/commands/external.rs (hunk 1 above)
Comment (rajatjindal) [reply src/commands/external.rs]: thank you ❤️

File: src/commands/external.rs (hunk 2 above)
Comment (rajatjindal) [reply src/commands/external.rs]: 💯  thank you

**Comment by rajatjindal** [comment]
thanks for the review Ivan. I have addressed the review comments. made a small change in error msg to include the plugin name explicitly. I hope that is ok.

**Review by itowlson — APPROVED** [review]
Looks grand now - thanks!

**Pull request merged**